lib/Onnxifi/InlineOnnxifi.cpp

```diff
@@ -22,14 +22,28 @@
 
 #include "llvm/Support/MD5.h"
 
+#ifdef WIN32
+#include <Lmcons.h>
+#include <windows.h>
+#else
 #include <unistd.h>
+#endif // !
 
 namespace glow {
 namespace onnxifi {
 
 namespace {
 std::string getProfileFile(llvm::StringRef hash) {
-  return strFormat("/tmp/glow-profile-%d-%s.yaml", getuid(),
+  std::string sUID;
+#ifdef WIN32
+  char username[UNLEN + 1];
+  DWORD username_len = UNLEN + 1;
+  GetUserName(username, &username_len);
+  sUID = username;
+#else
+  sUID = getuid();
+#endif
+  return strFormat("/tmp/glow-profile-%s-%s.yaml", sUID.c_str(),
```

Comment:
question. Do we actually need a user id here?

Reply:
@rdzhabarov I moved this code without trying to change it as much as possible. I agree that this seems unnecessary and would rather see it go.

Reply:
Okay. If we can get rid of getuid and GetUserName() that would be great. It would simplify the code significantly. Let's do it. 

Sorry for the delay @ayermolo .

Reply:
No problem. Is there an ETA for this fix?

Reply:
https://github.com/pytorch/glow/pull/2635

fixed here.

Reply:
Awesome, thanks!